examples/notebooks/documentation/transforms/python/elementwise/pardo-py.ipynb

```diff
@@ -23,7 +23,10 @@
    "execution_count": null,
    "metadata": {
     "cellView": "form",
-    "id": "_-code"
+    "id": "_-code",
+    "vscode": {
+     "languageId": "python"
+    }
```

Comment:
are these changes intentional?

Reply:
Ah I guess vscode added them when it was used for find/replace. I suppose that's harmless.

Reply:
Yeah they probably are, but I removed them anyways since they're not helpful